src/connector/src/sink/iceberg.rs

```diff
@@ -201,8 +207,7 @@ impl IcebergSink {
 
 #[async_trait::async_trait]
 impl Sink for IcebergSink {
-    /// TODO: Can't support commit coordinator now.
-    type Coordinator = DummySinkCommitCoordinator;
+    type Coordinator = IcebergSinkCommitter;
     type Writer = IcebergWriter;
 
     async fn validate(&self, _client: Option<ConnectorClient>) -> Result<()> {
```

Comment:
This method is for validating the user input props and will be called before the sink is created. I think we should check whether table exists and table schema and do other validation here.